Dockerfile

```diff
@@ -5,7 +5,7 @@
 # STAGE 1: Official PEP 513 Python Manylinux (RHEL5) base with Python 3.4 enabled to create
 #          linux_x86_64 pex.
 #          NOTE: To update image run: "docker pull quay.io/pypa/manylinux1_x86_64:latest"
-FROM quay.io/pypa/manylinux1_x86_64@sha256:1fc8ee3bc9d668222a519d84d8c3b3e2bbc3eabe4595a1661d7b037a038d2e87 AS stage1
+FROM quay.io/pypa/manylinux1_x86_64:latest AS stage1
```

Comment:
Why are you using latest instead of pinning it to a version?

Reply:
See my other comment.